src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/OffenderBookingRepository.java

```diff
@@ -71,4 +72,12 @@ List<OffenderBookingId> findDistinctByActiveTrueAndOffenderNomsIdInAndSentences_
     @Override
     @EntityGraph(type = EntityGraphType.FETCH, value = "booking-with-summary")
     Page<OffenderBooking> findAll(@NotNull Specification<OffenderBooking> filter, @NotNull Pageable pageable);
+
+    @Query("""
+    SELECT new uk.gov.justice.hmpps.prison.api.model.LatestTusedData(sc.tusedCalculatedDate, sc.tusedOverridedDate, sc.comments, o.nomsId)
+    from SentenceCalculation sc
+    inner join sc.offenderBooking.offender o where o.nomsId = :nomsId and coalesce(sc.tusedOverridedDate, sc.tusedCalculatedDate) is not null
```

Comment:
seems a slightly odd way of doing it - if the latest booking doesn't have a TUSED date then it will use a previous booking values? Is that correct?

Reply:
Yes, it's peculiar to this sort of sentence that we're looking at, need to consider dates from previous bookings that might still be valid

Reply:
probably worth adding something to the endpoint swagger docs describing ☝️ since it could be returning data people aren't expecting